<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>기록하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="breakfast_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ink Free</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>아침</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lunch_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>점심</string>
   </property>
  </widget>
  <widget class="QPushButton" name="latenight_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>취침 전</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dinner_button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>저녁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="record_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>480</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>기록보기</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>341</width>
     <height>236</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="plot_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
